packages/synapse-interface/components/Portfolio/PortfolioTokenAsset.tsx

```diff
@@ -39,19 +45,62 @@ const handleFocusOnInput = () => {
   inputRef.current.focus()
 }
 
+function checkCCTPChainConditions(
+  fromChainId: number,
+  toChainId: number
+): boolean {
+  switch (true) {
+    case fromChainId === 1 && toChainId === 42161:
```

Comment:
```
import {
  ARBITRUM,
  AVALANCHE,
  ETH,
} from '@/constants/chains/master'
```

```
case fromChainId === ETH.id && toChainId === ARBITRUM.id
...
```

for consistency w rest of app

Reply:
good catch! will have to rem to use our chain constants when comparing cId